Makefile

```diff
@@ -0,0 +1,10 @@
+CFLAGS ?= -O3
+
+bf: bf.o
+	${CC} -o bf bf.o
+
+bf.o: bf.c
+	${CC} ${CFLAGS} -Wall -c bf.c
```

Comment:
May as well add LDFLAGS?

Reply:
Sort of fixed in 6f3e369 -- I added it to the linking line, not the compiling line.

Reply:
Yeah, that's what I meant. My bad.

Reply:
In fact, you could use a suffix rule for making the .o, but I won't lose sleep about it :P

Reply:
I *think* later versions of this might need different flags but I'm not sure.

Reply:
Ah yes. You will need a barrage of odd LTO and section flags.